### PR TITLE
Feature/bb 3366 save documents on project

### PR DESCRIPTION
**Pull request by eodolphi**

### CODE REVIEW

File: bluebottle/projects/dashboard.py [hunk 1]
```diff
@@ -38,6 +38,7 @@ def init_with_context(self, context):
         qs = qs.filter(status=ProjectPhase.objects.get(slug="plan-submitted"))
 
         self.children = qs[:self.limit]
+        import ipdb; ipdb.set_trace()
```
Comment (gannetson) [inline bluebottle/projects/dashboard.py]: Free beer for the entire team!!

File: bluebottle/organizations/migrations/0006_auto__del_organizationdocument.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
```
Comment (gannetson) [inline bluebottle/organizations/migrations/0006_auto__del_organizationdocument.py]: Shouldn't this depend on project migrations?

File: bluebottle/projects/dashboard.py (hunk 1 above)
Comment (eodolphi) [inline bluebottle/projects/dashboard.py]: Oops...

File: bluebottle/organizations/migrations/0006_auto__del_organizationdocument.py (hunk 2 above)
Comment (eodolphi) [inline bluebottle/organizations/migrations/0006_auto__del_organizationdocument.py]: Do be honest, I did not know that was possible. But: Fixed.

**Pull request merged**